src/Pages/ProductDetail/Components/Shipping.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from "react";
+
+export class Shipping extends Component {
```

Comment:
마찬가지로 하단에서 export 해주시면
```suggestion
class Shipping extends Component {
```
이런식으로 작성해주세요!